### PR TITLE
NV: deals with petition bills and a few other tweaks

### PR DESCRIPTION
**Pull request by rshorey**
@mileswwatkins will have a field day that I willingly used a regex

### CODE REVIEW

**Comment by mileswwatkins** [comment]
All my regexes live in Texas, that's why I'm in Tennessee.

**Comment by mileswwatkins** [comment]
[`from re import funtimez`](http://funfunfunfunfunfunfunfun.funfunfunfunfunfunfunfun.com/)

File: openstates/nv/bills.py [hunk 1]
```diff
@@ -245,8 +245,18 @@ def scrape_actions(self, root, bill, actor):
                         action_type = atype
                         break
 
+
+                if "Committee on" in action:
+                    committees = re.findall("Committee on ([a-zA-Z, ]*)\.",action)
```
Comment (mileswwatkins) [inline openstates/nv/bills.py]: What if there's text _after_ the committee name? How do you know when to stop the `re` search?

**Comment by mileswwatkins** [comment]
That's my only comment. Still can't run locally, which is a pain, so I can't look at the data directly, unfortunately.

File: openstates/nv/bills.py (hunk 1 above)
Comment (rshorey) [inline openstates/nv/bills.py]: That's what the literal period is for! I checked a bunch and they all had extra text, but the committee was always the end of a sentence.

**Comment by rshorey** [comment]
I did some additional data check since you can't see them, and saw no errors, so I'm good with a merge if you are.

**Pull request merged**